.devcontainer/docker-compose.yml

```diff
@@ -1,29 +1,68 @@
-version: '3.1'
+version: "3.7"
 
 services:
   app:
-    image: wordpress
+    image: ghcr.io/ndigitals/wp-dev-container:php-8.2-node-20
```

Comment:
Note sure we'd want to use any third-pary image here that's not maintained by WP or Docker 🤔

Reply:
There is no prebuilt Docker container that exists that includes everything in this prebuilt Dev Container image that is also built for multi architecture. I have spent time to specifically build this image based off of the WP Dev Docker image. The entire source for it is open for review as well. If you want to force Devs that want to use the Dev Container on Apple chips to have slow subpar, or potentially broken, experiences then you can use the WP Dev image, however it's also going to require significantly longer startup time for Devs as well.

Reply:
What would need to added to the WordPress Dev contains in order to support this?